pkg/accumulator/accumulator_test.go

```diff
@@ -0,0 +1,34 @@
+package accumulator
+
```

Comment:
Any test fixtures/binaries should be placed under `testdata` directory. Go tooling knows to ignore anything placed under this directory. I'm referring to `kernel_obj_mock.o` & `userspace_app_mock`.
In addition, I don't see where it is being used.

Reply:
please create a `testdata` directory under the `config` directory and move both files there. The convention is that the testdata directory is in the same directory of its consumer.